setup.py

```diff
@@ -64,8 +64,8 @@
             'flake8-docstrings >=1.7.0, < 2.0',
             'flake8-print >=5.0.0, < 6.0',
             'flake8-rst-docstrings >=0.3.0, < 0.4.0',
-            'openfisca-country-template >=6.0.2 , <7 ',
-            'pytest >=7.2.2, < 8.0',
+            'openfisca-country-template @ https://github.com/openfisca/country-template/archive/chore/update-core-42.0.0.zip',
```

Comment:
Warning : Packages installed from PyPI cannot depend on packages which are not also hosted on PyPI.

So this will block PyPi publication of openfisca-survey-manager

Reply:
This is just to test that changes to core do not break this lib. Once the corresponding PR is accepted and merged, this has to be updated to use the regular PyPi package.